server/index.js

```diff
@@ -14,11 +14,31 @@ app.use(sessionMiddleware);
 app.use(express.json());
 
 app.get('/api/health-check', (req, res, next) => {
-  db.query(`select 'successfully connected' as "message"`)
+  db.query('select \'successfully connected\' as "message"')
     .then(result => res.json(result.rows[0]))
     .catch(err => next(err));
 });
 
+app.get('/api/product-check', (req, res, next) => {
```

Comment:
When making API routes, we typically make them plural nouns (very similar to database tables). My `health-check` example is just there for testing purposes to make sure that the starter code "works".